### ISSUE TITLE
Recommendation Algorithm

### ISSUE THREAD

**Issue by wkqzxh** [wkqzxh/Record-Beautiful-Lifle#5]
about rec algo

**Comment by wkqzxh** [comment]
**强化学习相关**
https://www.jiqizhixin.com/articles/2018-08-03-2
https://yq.aliyun.com/articles/414586
https://juejin.im/entry/58b2e30cac502e0069d9a3fa
https://medium.com/@yaoyaowd/%E9%98%BF%E9%87%8C%E5%92%8C%E4%BA%AC%E4%B8%9C%E7%9A%84%E5%A2%9E%E5%BC%BA%E5%AD%A6%E4%B9%A0-%E8%83%BD%E6%AF%94learning-to-rank%E5%A5%BD%E4%B8%80%E5%80%8D-a779e3a4cd65
https://yq.aliyun.com/articles/108481

**Comment by wkqzxh** [comment]
阿里短视频推荐
https://mp.weixin.qq.com/s/lb5b-7ImTI0hlFwIBkpqxQ

爱奇艺个性化推荐排序实践
http://www.6aiq.com/article/1525103316977?p=1&m=0

美团推荐算法：机器学习重排序模型
https://blog.csdn.net/bigheadyushan/article/details/77619393

微信“看一看”推荐模型
https://www.infoq.cn/article/H_QCTELLCkexaYv5Im5j

知乎推荐页 Ranking 构建历程和经验分享
https://www.infoq.cn/article/g95hu67a4WheikGu*w9K

今日头条推荐算法
https://www.iamue.com/35629?utm_source=joyk.com&utm_medium=referral&hmsr=joyk.com

Netflix推荐系统模型的快速线上评估方法——Interleaving
https://flashgene.com/archives/43759.html

**Comment by wkqzxh** [comment]
Batch Normalization原理与实战
https://zhuanlan.zhihu.com/p/34879333
https://zhuanlan.zhihu.com/p/43200897

TensorFlow实战——个性化推荐
https://zhuanlan.zhihu.com/p/32078473

attention is all you need
https://zhuanlan.zhihu.com/p/34465668

tensorflow wide and deep 模型实践
https://blog.csdn.net/cqlboat/article/details/82288072#demo-%E4%B8%BB%E8%A6%81%E6%BA%90%E7%A0%81%E5%88%86%E6%9E%90

完全图解RNN、RNN变体、Seq2Seq、Attention机制
https://zhuanlan.zhihu.com/p/28054589
